SonarQube.Client/Models/ServerSentEvents/SSEStreamReader.cs

```diff
@@ -92,19 +92,13 @@ private async Task<ISqServerEvent> ReadNextEventAsync()
         {
             try
             {
-                var sqEvent = await sqSSEStreamReader.ReadAsync();
-
-                return sqEvent;
-            }
-            catch (TaskCanceledException)
-            {
-                return null;
+                return await sqSSEStreamReader.ReadAsync();
             }
             catch (Exception ex)
             {
-                logger.Debug($"[SSEStreamReader] Failed to read sq event: {ex}");
-
-                return null;
+                logger.Warning($"[SSEStreamReader] Underlying stream failure while reading next event: {ex}");
```

Comment:
Logging: we need to co-ordinate the logging between the web client and SLVS so we don't overwhelm the users with warnings (less important for debug messages since they are off by default). Just something to bear in mind when we make the changes on the SLVS side.

Reply:
I felt like we should log this closer to the source. We still can log it on slvs side as well, maybe in less detail or in debug instead

Reply:
IMO it's better not to do anything here and to let the errors propagate to SLVS.
SLVS will then log, cancel and dispose everything.

Reply:
It's a transport layer error and I feel like it should be logged at the transport layer. Otherwise, what's the point of having the Error/Warn logger in SQS at all, if the only purpose of it is to write Debug logs?

Reply:
You should enable our debug logs and try to see what happens... ;) we already log way too much, adding another log won't provide any additional information and will just clutter it further